cli/app.go

```diff
@@ -425,6 +427,35 @@ var app = &cli.App{
 			Usage:           "work with organizations",
 			HideHelpCommand: true,
 			Subcommands: []*cli.Command{
+				{
+					Name:  "auth-service",
+					Usage: "manage auth-service",
+					Subcommands: []*cli.Command{
+						{
+							Name:  "oauth-app",
+							Usage: "manage the OAuth applications for an organization",
+							Subcommands: []*cli.Command{
+								{
+									Name:  "read",
```

Comment:
Why are we changing `get` to `read` for pulling the config for an OAuth application. Copied this from the scope:
```
"viam organization auth-service oauth-app create / read / update / delete " is a very logical replacement for "*viam auth-app register / update / get*". From a UX perspective, it is cleaner to create oauth-apps within the auth-service.
```

The other changes make sense to me but `read` doesn't seem either industry or viam standard to encapsulate making a GET request for an object. 

We use `get` for logo / support email and `get-config` for branded billing, and it seems the standard for our CLI is `get` or just the name of the object being returned (i.e. `robot logs` returns the robot logs) . I've  also never seen `read` an alias in other SDKs (happy to be proven wrong here) but this alias doesn't make logical sense to me. 

Something like `config` to show that we return the config for the auth application makes a lot more sense to me. 

@maxhorowitz @ohEmily